app/views/helpers/InlineDateInput.scala.html

```diff
@@ -31,7 +31,7 @@
 <fieldset class="form-group form-date @fieldsetClasses" id="@fieldName-fieldset">
 
     @if(questionText.isDefined) {
-        <legend><span style="display: none">@questionText.get</span></legend>
+        <legend><span class="visually-hidden">@questionText.get</span></legend>
```

Comment:
I'd just add the class to the legend and get rid of the span. Same for the RevealingRadios helper.